server/liveblog/syndication/utils.py

```diff
@@ -139,48 +134,64 @@ def extract_post_items_data(original_doc):
         "_updated",
     )
 
+    users_service = get_resource_service("users")
+    original_creator = users_service.find_one(req=None, _id=doc["original_creator"])
+
+    if not original_creator:
+        return None
+
+    creator_data = {}
+    for key in needed_fields:
+        if key in original_creator:
+            creator_data[key] = original_creator.get(key)
```

Comment:
Suggestion to use comprehension: 

`creator_data = {key: original_creator[key] for key in needed_fields if key in original_creator}`

Reply:
I'm not a huge fan of `comprehension` because I think they do not contribute much in terms of readability but, your suggestion makes sense from the "performance" point of view. Nevertheless, given that the amount of data we process here is really small, I'd argue that the performance differences would be negligible therefore I would be more inclined into readability over performance.

What do you think?

Reply:
I think then it's fine then if the amount of data processed here is small. That's ok with me